src/gcp/cloudfunctions.js

```diff
@@ -94,6 +94,37 @@ function _createFunction(options) {
     );
 }
 
+/**
+ * Sets the IAM policy of a Google Cloud Function.
+ * @param {*} options Options object.
+ * @param {string} options.projectId Project that owns the Function.
+ * @param {string} options.region Region in which the Function exists.
+ * @param {string} options.functionName Name of the Function.
+ * @param {*} options.policy The [policy](https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions/setIamPolicy) to set.
+ */
+async function _setIamPolicy(options) {
+  const name = `projects/${options.projectId}/locations/${options.region}/functions/${
+    options.functionName
+  }`;
+  const endpoint = `/${API_VERSION}/${name}:setIamPolicy`;
+
+  try {
+    await api.request("POST", endpoint, {
+      auth: true,
+      data: {
+        policy: options.policy,
+        updateMask: Object.keys(options.policy).join(),
```

Comment:
TIL that `join()` without an argument defaults to comma. That's a little too esoteric for my taste, can we make it explicit?

Reply:
Sure, I can do that.